test/lnd_services_mock.go

```diff
@@ -10,7 +10,6 @@ import (
 	"github.com/btcsuite/btcd/wire"
 	"github.com/lightninglabs/lndclient"
 	"github.com/lightningnetwork/lnd/chainntnfs"
-	"github.com/lightningnetwork/lnd/input"
```

Comment:
nit: could be a separate commit

Reply:
I think we generally want each commit to compile on its own. And since these changes were necessary because of the `lndclient` update, they should be in the same commit.

Reply:
Yeah, wanted it to compile.